torch/csrc/jit/codegen/cuda/ir_interface_nodes.h

```diff
@@ -469,7 +469,7 @@ class TORCH_CUDA_CU_API TensorView : public Val {
   void setDomain(TensorDomain* td) {
     domain_ = td;
   }
-
+public:
```

Comment:
Is this intetional?

Reply:
Oh, was this copied from my tuning change?

Reply:
Oh yeah, let me revert that, was testing with it, didn't mean to grab this change.